packages/k2-alpine/src/theme/theme.ts

```diff
@@ -9,7 +9,7 @@ export const darkTheme = makeTheme({
   },
   colors: darkModeColors,
   text,
-  isDark: true as boolean
+  isDark: true
```

Comment:
this will fail tsc if I remember correctly

Reply:
reverted.